tools/github-action/pack.go

```diff
@@ -26,54 +28,30 @@ func (opts *GithubAction) packAndPush(ctx context.Context) error {
 		format = "oci"
 	}
 
-	var err error
-	pm := packmanager.G(ctx)
-
-	// Switch the package manager the desired format for this target
-	if format != "auto" {
-		pm, err = pm.From(format)
-		if err != nil {
-			return err
-		}
-	}
-
-	// If no arguments have been specified, use the ones which are default and
-	// that have been included in the package.
-	if len(opts.Args) == 0 {
-		if len(opts.project.Command()) > 0 {
-			opts.Args = opts.project.Command()
-		} else if len(opts.target.Command()) > 0 {
-			opts.Args = opts.target.Command()
-		}
-	}
-
-	cmdShellArgs, err := shellwords.Parse(strings.Join(opts.Args, " "))
+	config.G[config.KraftKit](ctx).Log.Type = "basic"
+	config.G[config.KraftKit](ctx).Log.Level = "info"
```

Comment:
We should not manipulate the log level as this is user programmable.

The log-type should automatically downgrade to `basic` (from `fancy`) in a non-TTY environment. If it is not downgrading, then there is an issue elsewhere.

Reply:
yeah it did not

The environment is a tty though unfortunately afaik :(